tests/saveConsoleLogs.test.ts

```diff
@@ -17,15 +18,14 @@ describe('saveConsoleLogs', () => {
   afterAll(() => browser.close());
 
   it('saves console logs to specified file', async () => {
-    const savePath = join(tmpdir(), `${Date.now()}.mp4`);
+    const savePath = join(tmpdir(), randomString(), `${Date.now()}.txt`);
```

Comment:
this prevents it from using the same `tmp` directory locally, which would cause new logs to be appended and the tests to fail on subsequent runs when running locally